torch/_dynamo/variables/tensor.py

```diff
@@ -348,10 +348,14 @@ def call_method(
                 unimplemented(f"Illegal method invocation {name} in strict mode")
         from . import ConstantVariable, TorchVariable, TupleVariable
         from .builder import wrap_fx_proxy
+        from .torch import is_torch_inplace_op
 
         kwargs = dict(kwargs)
         options = VariableTracker.propagate(self, args, kwargs.values())
 
+        if is_torch_inplace_op(name):  # inplace / self-propagating op
+            options.update({"source": self.source})
```

Comment:
This method is no longer viable. Please use `FakeTensor` identity instead. Fixing the higher-order ops issue is another matter.